app/services/puppet_fact_parser.rb

```diff
@@ -6,8 +6,8 @@ def operatingsystem
 
     if orel.present?
       if os_name =~ /ubuntu/i
-        major = os_major_version
-        minor = os_minor_version
+        major = os_major_version.to_s
+        minor = os_minor_version.to_s
```

Comment:
obviously the `.to_s` can also directly in those functions, I have no preference.